powerauth-webauth-authentication/src/main/java/io/getlime/security/powerauth/lib/webauth/authentication/configuration/WebAuthServicesConfiguration.java

```diff
@@ -57,7 +81,44 @@ public CredentialStoreClient defaultCredentialStoreClient() {
      */
     @Bean
     public NextStepClient defaultNextStepClient() {
-        return new NextStepClient(nextstepServiceUrl);
+        NextStepClient client = new NextStepClient(nextstepServiceUrl);
+        // whether invalid SSL certificates should be accepted
+        if (acceptInvalidSslCertificateInNextStep) {
+            trustAllCertificates();
+        }
+        return client;
     }
 
+    /**
+     * Activate trust in all SSL certificates including invalid ones for non-production use.
+     */
+    private void trustAllCertificates() {
```

Comment:
This method disables SSL certificate handling everywhere - not just on one or another interface. We should either:

- implement SSL disabling in such way that it works per client (Next Step vs. Credential Store), or...
- have only one property "acceptInvalidSslCertificate" that disables SSL check for all services.